### DISCUSSION TITLE
bad translation for 'dependency graph' in portuguese

### DISCUSSION THREAD

**Discussion by Barbiero** [community/community#16652] [Code Security]
Page:
PT-BR https://docs.github.com/pt/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph
EN https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph

Graph is a word that can be translated to "Gráfico" or "Grafo" depending on context. When "Graph" refers to a mathematical plot(like the ones with x,y axis), it is translated as "Gráfico". `Gráfico` is heavily correlated to statistics. Think of "Graphics".

Meanwhile, the mathematical structure that represents vertices and edges is called `Grafo`. A Dependency Graph is os this type, and so this whole page should be reviewed to use the phrase "Grafo de Dependências" instead of "Gráfico". Right now, it feels like it is talking about some graphical representation for statistical dependencies.

**Comment by courtneycl** [comment]
Thank you for the thoughtful feedback, @Barbiero! I've passed this to our docs team.
